### PR TITLE
Add Inkscape v1.0.1

### PR DESCRIPTION
**Pull request by brechtcallens**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/3480)

### CODE REVIEW

**Comment by brechtcallens** [comment]
> 
> 
> brechtcallens The package manager bot determined there was an issue with one of the installers listed in the url field, and cannot continue.
> 
> The installer has been identified as malware. Please check that the installer URL is correct and update the URL and Hash if a change is made.

How is Inkscape malware? It's downloadable on their official website: https://inkscape.org/release/1.0.1/windows/64-bit/

File: manifests/Inkscape/Inkscape/1.0.1.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Publisher: The Inkscape Project
+License: GPL 3.0 or later
+LicenseUrl: https://inkscape.org/about/license/
+AppMoniker: inkscape
+Tags: inkscape, editor, foss, cross-platform, svg, vector-graphics, icons, graphics, export, drawing, art
+Description: Professional vector graphics editor
+Homepage: https://inkscape.org/
+Installers:
+  - Arch: x64
+    Url: https://inkscape.org/gallery/item/21592/inkscape-1.0.1-x64.msi
```
Comment (yigitemres) [inline manifests/Inkscape/Inkscape/1.0.1.yaml:13]:
```suggestion
    Url: https://media.inkscape.org/dl/resources/file/inkscape-1.0.1-x64.msi
```
This is the direct link to the installer.

**Review by yigitemres — CHANGES_REQUESTED** [review]
@brechtcallens Maybe redirection cause something stupid and cause malware alert?

File: manifests/Inkscape/Inkscape/1.0.1.yaml (hunk 1 above)
Comment (brechtcallens) [reply manifests/Inkscape/Inkscape/1.0.1.yaml:13]: Ok, thanks! I'll try it.

**Pull request merged**